Detector/DetFCChhECalDigital/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+################################################################################
+# Package: DetFCChhECalDigital
+################################################################################
+gaudi_subdir(DetFCChhECalDigital v1r0)
+
+gaudi_depends_on_subdirs(GaudiKernel
+                        Detector/DetExtensions)
+
+
+find_package(DD4hep)
+find_package(Geant4)
+include(${Geant4_USE_FILE})
+
+set(CMAKE_MODULE_PATH  ${CMAKE_MODULE_PATH}  ${DD4hep_ROOT}/cmake )
+include( DD4hep )
+
+find_package(ROOT COMPONENTS MathCore GenVector Geom REQUIRED)
+
```

Comment:
Please add line:
`install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/compact DESTINATION Detector/DetFCChhECalDigital)`
so the detector can be used from the pre-installed FCCSW.